cmd/buildah/commit.go

```diff
@@ -77,6 +77,10 @@ var (
 			Name:  "tls-verify",
 			Usage: "Require HTTPS and verify certificates when accessing the registry",
 		},
+		cli.BoolFlag{
+			Name:  "omit-timestamp",
+			Usage: "Set createdBy timestamp to epoch 0 to allow for deterministic builds",
```

Comment:
s/createdBy/created/   createdBy is a string representing who/what created the image.  'created' is the time stamp.

Reply:
Also, should we make this '-o' also?

Reply:
thanks for spotting this one. of course, created**By** is completely wrong. i changed it to created.
about the -o : i'm not sure if this is so frequently used that such a short flag should be used for it. what do you think?

Reply:
No this is not used enough to require a short option.